### PR TITLE
Uncommented element that previously contained desired text.

### PR DESCRIPTION
**Pull request by SQrL143**

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -359,7 +359,16 @@
       "contributions": [
         "code"
       ]
-    }
+    }    {
```
Comment (csell5) [inline .all-contributorsrc:362]: Roll this out.. I have another process that will set this correctly.

**Review by csell5 — CHANGES_REQUESTED** [review]
Just one minor change and make sure you're rebased on master.

File: .all-contributorsrc (hunk 1 above)
Comment (SQrL143) [reply .all-contributorsrc]: See new commits. JSon formatting fixed. I also moved the location of the text so that there is better visual contrast.

Comment (SQrL143) [reply .all-contributorsrc]: @csell5 FYI

Comment (csell5) [reply .all-contributorsrc]: I'm sorry.. I meant roll that whole block out as we have a bot that will get you setup.

**Comment by csell5** [comment]
@all-conttributors please add @SQrL143 for code.

**Comment by csell5** [comment]
@all-contributors please add @SQrL143 for code.

File: .all-contributorsrc (hunk 1 above)
Comment (csell5) [reply .all-contributorsrc]: ok, I ran the bot and got that file sorted out. pull that out and rebase again and it should be all wired right.

Comment (SQrL143) [reply .all-contributorsrc]: @csell5 I pushed up a change that removed my profile from allcontributors in my branch so I think it will merge correctly. I

**Comment by csell5** [comment]
thank you @SQrL143 sorry about the hassle there.

**Comment by SQrL143** [comment]
I am also sorry for the confusion

**Review by csell5 — APPROVED** [review]

**Pull request merged**